aten/src/ATen/native/cuda/ForeachBinaryOpScalar.cu

```diff
@@ -191,11 +191,44 @@ std::vector<Tensor> foreach_scalar_pow_list_kernel_cuda(
 // In the case of division, integer inputs will result in float.
 // Currently multi tensor apply can only return result of the same type as
 // input.
-FOREACH_BINARY_OP_SCALAR(
-    all_types_complex_bool_half_bfloat16,
-    div,
-    std::divides,
-    /*div_op*/ true);
+//
+// Implement via multiply with reciprocal as it's faster.  Loses one bit of
+// precision.
```

Comment:
nit: Do you think it is worth mentioning that this is the same behavior as `torch.div_`?